Core/Pixel.swift

```diff
@@ -134,6 +134,13 @@ public enum PixelName: String {
     case feedbackNegativePerformanceVideo = "mfbs_negative_performance_video"
     case feedbackNegativePerformanceOther = "mfbs_negative_performance_other"
     
+    case notificationOptIn = "m_ne"
+    case notificationOptOut = "m_nd"
+    case notificationD0Fired = "m_nfi"
+    case notificationD1Fired = "m_nfii"
+    case notificationD0Opened = "m_noi"
+    case notificationD1Opened = "m_noii"
+    
```

Comment:
These notifications are described as "privacy" and "home row" elsewhere.  Would it be better to name these similarly for clarity?